api/backend.go

```diff
@@ -236,7 +235,25 @@ func (b *StatusBackend) StartNodeWithKey(acc multiaccounts.Account, password str
 		return err
 	}
 	b.accountManager.SetAccountAddresses(chatAcc.Address)
-	return b.injectAccountIntoServices()
+	err = b.injectAccountIntoServices()
+	if err != nil {
+		return err
+	}
+	err = b.multiaccountsDB.UpdateAccountTimestamp(acc.Address, time.Now().Unix())
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (b *StatusBackend) StartNodeWithKey(acc multiaccounts.Account, password string, keyHex string) error {
+	err := b.startNodeWithKey(acc, password, keyHex)
+	signal.SendLoggedIn(err)
```

Comment:
Not sure how significant it is but a potential scenario looks like this :
1. `StartNodeWithKey` is called in one thread. `startNodeWithKey` returns an error.
2. A signal is received in a different thread.
3. status-react tries to restart node while `b.StopNode()` is not finished.

Should be move `signal.SendLoggedIn(err)` behind `b.StopNode()`?

Reply:
status-react only starts a nice node via login though, but we could still do that, it would have do be node in StartNodeWithAccount as well though, that is where I took this flow from